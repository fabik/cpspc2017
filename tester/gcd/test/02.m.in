15
26 23 7 9 15 12 16 9 25 9 2 20 8 16 17
