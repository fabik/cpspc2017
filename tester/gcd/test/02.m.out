1
27
52
75
95
112
128
144
159
171
180
189
198
206
214
